### PR TITLE
Implemented native method for CoreSchema#getActualConfiguration [ECR-2646].

### PR DESCRIPTION
**Pull request by skletsun**
## Overview
The method returns the configuration for the latest height of the blockchain serialized to JSON string.

---
See: https://jira.bf.local/browse/ECR-2646

### Definition of Done

- [x] There are no TODOs left in the code
- [ ] Change is covered by automated [tests](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#tests)
- [x] The [coding guidelines](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#the-code-style) are followed
- [x] Public API has Javadoc
- [x] Method preconditions are checked and documented in the Javadoc of the method
- [ ] Changelog is updated if needed (in case of notable or breaking changes)
- [x] The continuous integration build passes

### CODE REVIEW

File: exonum-java-binding-core/rust/src/storage/core_schema.rs [hunk 1]
```diff
@@ -94,3 +94,25 @@ pub extern "system" fn Java_com_exonum_binding_blockchain_CoreSchemaProxy_native
     });
     utils::unwrap_exc_or(&env, res, ptr::null_mut())
 }
+
+/// Returns the configuration for the latest height of the blockchain. Throws
+/// `java.lang.RuntimeException` if the "genesis block" has not been created yet.
```
Comment (DENMROOT) [inline exonum-java-binding-core/rust/src/storage/core_schema.rs]: Can we throw custom exception here?  RuntimeException is too general.

File: exonum-java-binding-core/rust/src/storage/core_schema.rs [hunk 2]
```diff
@@ -94,3 +94,25 @@ pub extern "system" fn Java_com_exonum_binding_blockchain_CoreSchemaProxy_native
...
+    env: JNIEnv,
+    _: JClass,
+    schema_handle: Handle,
+) -> jstring {
+    let res = panic::catch_unwind(|| {
+        let val = match utils::cast_handle::<SchemaType>(schema_handle) {
+            SchemaType::SnapshotSchema(schema) => schema.actual_configuration(),
+            SchemaType::ForkSchema(schema) => schema.actual_configuration(),
+        };
+        serde_json::to_value(val)
```
Comment (vitvakatu) [inline exonum-java-binding-core/rust/src/storage/core_schema.rs]: This is redundant, you should be able to use `serde_json::to_string` directly, without conversion to `serde_json::Value`

File: exonum-java-binding-core/rust/src/storage/core_schema.rs (hunk 1 above)
Comment (skletsun) [reply exonum-java-binding-core/rust/src/storage/core_schema.rs]: I agree that it's rather general, but for the moment it's our convention. We can consider it as an area for further improvement.

File: exonum-java-binding-core/rust/src/storage/core_schema.rs (hunk 2 above)
Comment (skletsun) [reply exonum-java-binding-core/rust/src/storage/core_schema.rs]: Thanks, corrected

**Pull request merged**